lib/migration.pm

```diff
@@ -119,7 +119,8 @@ sub disable_installation_repos {
 # Record disk info to help debug diskspace exhausted
 # issue during upgrade
 sub record_disk_info {
-    if (get_var('FILESYSTEM', 'btrfs') =~ /btrfs/) {
+    my $ret = script_run 'mount | grep btrfs | grep "on / "';
+    if ($ret =~ /btrfs/) {
```

Comment:
if I remember correctly script_run returns exit status of command - not output

Reply:
Yes, thanks for pointing the wrong using. I changed it to script_output. 
Verify log: http://openqa-apac1.suse.de/tests/4521#